### DISCUSSION TITLE
AMS 2022 abstract - Sept 1 deadline!!!

### DISCUSSION THREAD

**Discussion by clyne** [ProjectPythia/projectpythia.github.io#144] [General]
All, we got so caught up in our discussion today that we didn't get around to talking about a submission for AMS. I'd like us to have a presence there given our current outreach efforts. There is a session on "Teaching, Training, Outreach, and Building Communities around Python" in the 12th Symposium on Advances in Modeling and Analysis Using Python. Perfect fit, right?

i've cobbled together a quick abstract [here](https://docs.google.com/document/d/1DYIjOhFECgT86ovZHkufsO1HOlUrX02e6IIUXF4-SF4/edit?usp=sharing). 

Please review/comment ASAP! 

If someone else would like to present - maybe from the OWG team (hint, hint) - that would be fine by me. 

Also, any reason to not simply use our GitHub contributors for the author list?

**Comment by brian-rose** [comment]
The deadline has been extended to Sep 8: https://annual.ametsoc.org/index.cfm/2022/
I can reserve some time at the EWG meeting this week to review the abstract together, although it already looks pretty good.

I am expecting to travel to the AMS meeting so I *can* give the presentation, but I'm happy to let someone else do it!

**Comment by clyne** [comment]
Sounds good. Let's discuss briefly on Thursday.
